src/one_dragon/envs/git_service.py

```
@@ -1,454 +1,656 @@
-                        continue
-                    else:
-                        # 稳定版先出现，则视为无测试版
-                        first_seen_type = 'stable'
-                        latest_stable = tag_name
-                        break
-
-                # 若先看到的是 beta，则继续找第一个稳定版
-                if first_seen_type == 'beta' and not is_beta and latest_stable is None:
-                    latest_stable = tag_name
+        """
+        # 如果不存在本地仓库，返回空
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)